util/custom-response/src/main/java/com/brother/customresponse/entity/Student.java

```diff
@@ -0,0 +1,22 @@
+package com.brother.customresponse.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import java.util.Set;
+
+@Setter
+@Getter
+@Entity
+public class Student extends BaseEntity {
+
+    private String name;
+    private String email;
+    @OneToMany(mappedBy = "student", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Can we use FetchType.LAZY instead of FetchType.EAGER?

Reply:
> Can we use FetchType.LAZY instead of FetchType.EAGER?

Done